app/views/standard/requestMetadataReview.scala.html

```diff
@@ -0,0 +1,31 @@
+@import java.util.UUID
+@import views.html.partials.transferReference
+@(consignmentId: UUID, consignmentRef: String, name: String, blockDraftMetadataUpload: Boolean)(implicit messages: Messages, request: RequestHeader)
+
+@defining("Request a metadata review") { title =>
+  @main(title, name = name) {
+    @defining(play.core.PlayVersion.current) { version =>
+      <<div class="govuk-grid-row">
+        <div class="govuk-grid-column-two-thirds-from-desktop">
+
+          <h1 class="govuk-heading-xl">@title</h1>
+
+          <p class="govuk-body">Before you complete your transfer your metadata needs to be reviewed by a Digital Transfer Advisor. This helps us to ensure we preserve your records accurately.</p>
+          <div class="govuk-inset-text">
+            If we find any issues with your metadata we will send guidance to <strong>jesse@@ons.gov.uk</strong>.
```

Comment:
Should the email address not be the user's, instead of the hard coded value: `jesse@@ons.gov.uk`

Reply:
Oh yeah. Sorry, I missed this one. Thanks I have updated it with the user's email